featuretools/entityset/entityset.py

```diff
@@ -1190,43 +1190,37 @@ def add_interesting_values(self, max_values=5, verbose=False, dataframe_name=Non
             dataframes = [self[dataframe_name]]
         else:
             dataframes = self.dataframes
+
+        def add_value(df, col, val, verbose):
+            if verbose:
+                msg = "Column {}: Marking {} as an "
+                msg += "interesting value"
+                logger.info(msg.format(col, val))
+            interesting_vals = df.ww.columns[col].metadata.get('interesting_values', [])
+            df.ww.columns[col].metadata['interesting_values'] = interesting_vals + [val]
```

Comment:
we could append instead of concat

Reply:
Hmmm. Might be missing something in your suggestion, but we need to handle the situation where we get a new list back from the `get` call in 1199. That new list (with the new value added) then needs to be assigned back to the metadata, which was previously not set. I think using append here would require an extra line of code, something like this:

```python
interesting_vals = df.ww.columns[col].metadata.get('interesting_values', [])
intesting_vals.append(val)
df.ww.columns[col].metadata['interesting_values'] = interesting_vals
```

Reply:
It would require an extra line of code but it would save the creation of an additional list by skipping the concat

Reply:
Sounds good. Happy to update, just wanted to make sure I understood your suggestion.